NRasterizer.Tests/OpenTypeReaderTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NRasterizer.Tests
+{
+
+    [TestFixture]
+    public class OpenTypeReaderTests
+    {
+        [Test]
+        [TestCaseSource(typeof(TestFontsPaths), "AllFonts")]
+        public void LoadingFontDoesntThrowAnyExceptions(string path)
+        {
+            var reader = new OpenTypeReader();
+            Assert.DoesNotThrow(() =>
+            {
+                using (var fs = File.OpenRead(path))
+                {
+                    reader.Read(fs);
+                }
+            });
```

Comment:
You can remove `DoesNotThrow` wrapping as test will fail anyway if an exception is thrown. This will just make it harder to find problem should this test fail.